handler/error.go

```diff
@@ -5,14 +5,14 @@ type Error struct {
 	logData map[string]interface{}
 }
 
-func (e *Error) Error() string {
+func (e Error) Error() string {
```

Comment:
Any particular reason you decided not to use a pointer receiver here?

Reply:
I imagine to get tests to pass when comparing error values?

Reply:
Yes, exactly. Do you think it's worth it?